models/staging/sources.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 sources:
   - name: raw_dbt_monitoring
     description: "Raw data extracted from the models generated by the elementary package."
-    schema: "{{ 'elementary' if target.name in ('prod') else (target.schema + '_elementary') }}"
+    schema: "{% if target.name == 'prod' %}elementary{% elif var('elementary_schema') %}{{ var('elementary_schema') }}{% else %}{{ target.schema ~ '_elementary' }}{% endif %}"
```

Comment:
I think this should be more direct and less "full of ifs"

schema = {{ var('elementary_schema') }}

The way youve proposed still we canot have a target prod that does not uses elementary schema. And I don't think it will be always called elementary schema

Reply:
Hmm, its true. But the package itself proposes this configuration above in the readme:

1. The package's models can be configured in your `dbt_project.yml` by specifying the package under `models`.

```
models:
    elementary:
        +schema: 'elementary'
```

So i've assumed that in prod, the target schema always will be called as elementary. What do you think about it?